.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Issue Template
+about: File an issue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+This is a suggested issue template for tedana.
+
+If there is other information that would be helpful to include, please don not hesitate to add it!
+
+Before submitting, please check to make sure that the issue is not already addressed.
+If this is a usage question, please checkout NeuroStars here:
+https://neurostars.org/
+and tag your topic with "multi-echo" so that we can see it.
+-->
+
+<!--
+Summarize the issue in 1-2 sentences, linking other issues if they are relevant
+
+Note: simply typing # will prompt you for open issues to select from
+-->
+### Summary
+
+<!--
+If needed, add additional detail for:
+1. Recreating a bug/problem
+2. Context
```

Comment:
Can we say "Any additional context necessary to understand the issue" ?

Reply:
Adjusted in 7937db4b4639faabc248c2c3192d7af1d26fdd45